src/translations/fa.ts

```diff
@@ -0,0 +1,26 @@
+import { registerTranslation } from '../utilities/localize';
+import type { Translation } from '../utilities/localize';
+
+const translation: Translation = {
+  $code: 'fa',
+  $name: 'Persian',
```

Comment:
This property isn't currently used, but could it be localized like the others? e.g. "Spanish" becomes "Español" and "French" becomes "Français".